lib/router-fetcher.js

```diff
@@ -1,31 +1,45 @@
 'use strict';
 
-const { promisify } = require('util');
-const request = promisify(require('request'));
+const axios = require('axios');
```

Comment:
Si agregamos esto, importamos los typeDef de Axios

```suggestion
const axios = require('axios').default;
```